CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-*                 @geeknoid @mandarjog @costinm @duderino
-/metrics          @mandarjog @costinm @howardjohn @utka
-/perf             @mandarjog @costinm @howardjohn @utka
-/protoc-gen-docs  @geeknoid
+*        @istio/wg-test-and-release-maintainers
+perf/    @istio/wg-perf-and-scalbility-maintainers
```

Comment:
I think the consensus was to make this perf+test. Otherwise it is basically saying only Mandar can approve, and mandar cannot approve his own changes so he can no longer make changes